libraries/AP_Networking/AP_Networking_Config.h

```diff
@@ -0,0 +1,140 @@
+#include <AP_HAL/AP_HAL_Boards.h>
+#include <GCS_MAVLink/GCS_config.h>
+
+#ifndef AP_NETWORKING_ENABLED
+#define AP_NETWORKING_ENABLED 0
+#endif
+
+#ifndef AP_NETWORKING_MAX_INSTANCES
+#define AP_NETWORKING_MAX_INSTANCES 4
+#endif
+
+#ifndef AP_NETWORKING_ETHERNET_TCP_PAYLOAD_MAX_SIZE
+#define AP_NETWORKING_ETHERNET_TCP_PAYLOAD_MAX_SIZE 1460 // 1500 (Eth MTU) - 20 (IP header) - 20 (TCP header)
+#endif
+#ifndef AP_NETWORKING_ETHERNET_UDP_PAYLOAD_MAX_SIZE
+#define AP_NETWORKING_ETHERNET_UDP_PAYLOAD_MAX_SIZE 1472 // 1500 (Eth MTU) - 20 (IP header) - 8 (UDP header)
+#endif
+
+#ifndef AP_NETWORKING_ETH_IP_UDP_OVERHEAD
+#define AP_NETWORKING_ETH_IP_UDP_OVERHEAD 51 // 23 (Eth frame) + 20 (IP header) + 8 (UDP header)
+#endif
+
+#ifndef AP_NETWORKING_SERIAL_ENABLED
+#define AP_NETWORKING_SERIAL_ENABLED AP_NETWORKING_ENABLED && HAL_GCS_ENABLED
+#endif
+
+#ifndef AP_NETWORKING_SPEEDTEST_ENABLED
+// Flash use: 1.5 KB
+#define AP_NETWORKING_SPEEDTEST_ENABLED AP_NETWORKING_ENABLED
+#endif
+
+#ifndef AP_NETWORKING_LATENCYTEST_ENABLED
+// Flash use: 568 Bytes
+#define AP_NETWORKING_LATENCYTEST_ENABLED 0
+#endif
+
+#ifndef AP_NETWORKING_SNTP_ENABLED
+// Flash use: 3.1 KB
+#define AP_NETWORKING_SNTP_ENABLED 0
+#endif
+
+#ifndef AP_NETWORKING_SNTP_SERVERNAME
+#define AP_NETWORKING_SNTP_SERVERNAME "pool.ntp.org"
+#endif
+
+
+#ifndef AP_NETWORKING_NETBIOS_ENABLED
+// Flash use: 800 Bytes
+#define AP_NETWORKING_NETBIOS_ENABLED 0
+#endif
+#ifndef AP_NETWORKING_NETBIOS_NAME
+    // This string must be 15 (NETBIOS_NAME_LEN) or less characters
+    #define AP_NETWORKING_NETBIOS_NAME "ARDUPILOT"
+#endif
+
+#ifndef AP_NETWORKING_TFTP_ENABLED
+// Flash use: 1.85 KB
+#define AP_NETWORKING_TFTP_ENABLED 0
+#endif
+
+
+#if CONFIG_HAL_BOARD == HAL_BOARD_CHIBIOS
+    #define AP_NETWORKING_DHCP_AVAILABLE    LWIP_DHCP
+#else
+    #define AP_NETWORKING_DHCP_AVAILABLE    1 // for non-ChibiOS, assume it's available
+#endif
+
+
+// ---------------------------
+// Below are default params
+// ---------------------------
+
+#ifndef AP_NETWORKING_DEFAULT_OPTIONS
+    #define AP_NETWORKING_DEFAULT_OPTIONS 0
+#endif
+
+// Default DHCP
+#ifndef AP_NETWORKING_DEFAULT_DHCP_ENABLE
+#define AP_NETWORKING_DEFAULT_DHCP_ENABLE 0
+#endif
+
+// Default Static IP Address: 192.168.13.13
```

Comment:
The comment disagrees with the values below, perhaps you can remove the comment?